bin/metababel

```diff
@@ -274,20 +274,16 @@ base_folder = options[:folder] || "#{options[:component_type]}.#{options[:plugin
 folder = "#{base_folder}/metababel"
 FileUtils.mkdir_p(folder)
 
-d = { options: options }
+d = { options: options, params_declaration: nil, params_definition: nil }
```

Comment:
I think this could be redundant, but I am not sure why did you add this

```ruby 
params_declaration: nil, params_definition: nil
```

Reply:
We need it, or we need a default dict.

Reply:
Ok, I understand, it is because the erb render requires the variable to be defined.

Reply:
Exactly. It's not defaulted to `nil` as usual.